docs/concepts/architecture.md

```diff
@@ -5,4 +5,12 @@ title: "Architecture"
 
 # Architecture
 
-TODO
+In the below diagram you can see the different layers of library and its dependencies.
```

Comment:
@khajavi  so the ZIO sbt `previewWebsite` task stopped working (website not running) and produces the following output in the logs (I am using the lasted version `0.4.0-alpha.28`)
```
> docusaurus start
sh: docusaurus: command not found
```
AFIAK management of docusaurus (eg install/setup) is done via the plugun. Any ideas?

Reply:
clean the target directory and try `sbt installWebsite` before running `sbt previewWebsite`
@googley42 Please let me know if it didn't work!

Reply:
yes that worked - many thanks!